test/update.test.js

```diff
@@ -144,7 +144,7 @@ describe('updateAll', function() {
       Product.update({id: id}, newData, function(err, result) {
         testUtil.hasError(err, result).should.not.be.ok();
         testUtil.hasResult(err, result).should.be.ok();
-        var newRev = result[0].rev;
```

Comment:
That is because `update` didn't work the correct way before. It would not convert the values back from the database correctly.